### PR TITLE
Perf API for Dart

### PR DESCRIPTION
**Pull request by marandaneto**
Perf API docs for https://github.com/getsentry/sentry-dart/releases/tag/6.1.0-alpha.1

### CODE REVIEW

**Review by philipphofmann — APPROVED** [review]
Good job, just a few minor comments 👏

File: src/includes/performance/default-sampling-context-platform/dart.mdx [hunk 1]
```diff
@@ -0,0 +1 @@
+For Dart-based SDKs, it includes a [SentryTransactionContext](https://pub.dev/documentation/sentry/6.1.0-alpha.1/sentry_io/SentryTransactionContext-class.html) and a [Map<String, dynamic>](https://api.dart.dev/stable/2.14.1/dart-core/Map-class.html) as method params for [Sentry.startTransactionWithContext](https://pub.dev/documentation/sentry/6.1.0-alpha.1/sentry_io/Sentry/startTransactionWithContext.html).
```
[the remark this answers is not in the document]
Comment (marandaneto) [reply src/includes/performance/default-sampling-context-platform/dart.mdx:1]: only when it goes out of the alpha version, for now, it should be specific

File: src/includes/performance/configure-sample-rate/dart.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Performance Monitoring is available for the Sentry Dart SDK version ≥ 6.1.0-alpha.1.
```
[the remark this answers is not in the document]
Comment (marandaneto) [reply src/includes/performance/configure-sample-rate/dart.mdx:1]: the setup page is a common page shared across all the SDKs, so we add to the include page that enables the feature(this one)

File: src/includes/performance/custom-sampling-context/dart.mdx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+```dart
```
[the remark this answers is not in the document]
Comment (marandaneto) [reply src/includes/performance/custom-sampling-context/dart.mdx:1]: content is the same, java has java and kotlin snippets, ios has obj c and swift, maybe that's the confusion?

File: src/includes/performance/distributed-tracing/dart.mdx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+To obtain a trace header from the span, use `ISentrySpan#toSentryTrace()` method. Then pass it to the downstream service. If the communication happens over HTTP, it is recommended that you set the value to the `sentry-trace` HTTP header.
```
Comment (bitsandfoxes) [inline src/includes/performance/distributed-tracing/dart.mdx:1]: `ISentrySpan#toSentryTrace()` is the `#` an actual thing?

Comment (marandaneto) [reply src/includes/performance/distributed-tracing/dart.mdx:1]: not really, on Java docs, its a convention that makes docs clickable, in Dart its [] but [] is also used in markdown, so I kept the Java way, I could do `ISentrySpan.toSentryTrace()` though, but it'd look like a static method but its not

**Review by bitsandfoxes — APPROVED** [review]

**Pull request merged**